pkg/controller/daemon/util/daemonset_util.go

```diff
@@ -115,3 +118,48 @@ func SplitByAvailablePods(minReadySeconds int32, pods []*v1.Pod) ([]*v1.Pod, []*
 	}
 	return availablePods, unavailablePods
 }
+
+// AppendDaemonSetPodNodeAffinity creates NodeAffinity entry for DaemonSet controller
+func AppendDaemonSetPodNodeAffinity(affinity *v1.Affinity, hostname string) *v1.Affinity {
```

Comment:
s/hostname/nodeName/
I know that our affinity key for node name is called "kubernetes.io/hostname", but that was a bit unfortunate. We should use "node" instead of "host". "Node" is the right word in K8s terminology.

Reply:
done :)